design_docs/rsyslog_ovirt_support.md

```diff
@@ -0,0 +1,133 @@
+# Rsyslog - oVirt Support
+
+## Background
+
+In the logging role context, oVirt is a special input type which handles inputs from the remote RHV systems and the files input (imfile) on the target host. The log messages are formatted in the ovirt input and forwarded to the elasticsearch. Plus, in the stable-0.1 version, it depended on the default rsyslog.conf which was deployed by setting the rsyslog_default variable to true. 
+
+The goal of the task "RHELPLAN-42379 - Logging - Support oVirt input" is providing the same logging experience using the logging_inputs, logging_outputs, and logging_flows configuration. Then, the rsyslog_default flag (with etc/rsyslog.conf.j2 template) and the send-target-only config file will be safely dropped.
+
+## Implementation Changes
+
+Note: in the effort of "RHELPLAN-41572 - Logging - Convert each subrole to a pair of task/var files", ovirt is also converted from a subrole to a pair of task/var files in roles/rsyslog/{tasks,vars}/inputs.
+
+### ovirt task
+
+Prior to this change made for "RHELPLAN-42379 - Logging - Support oVirt input", ovirt role passed all the __rsyslog_ovirt_rules defined in the [vars main.yml](../roles/rsyslog/vars/inputs/ovirt/main.yml) to [deploy.yml](../roles/rsyslog/tasks/deploy.yml). The __rsyslog_ovirt_rules are devided into 2 groups - static and dynaimc. The static group is processed in the same way as before. The dynamic group handles the forwarding to the outputs based on the logging_flows.
```

Comment:
divided into 2 groups

Reply:
static and dynamic

Reply:
Thank you, @richm. I've installed and ran aspell to find lots of misspellings...  I promise I'll check my docs before pushing. (;_;)